dev/scripts/docgen/api/keywordTable.ts

```diff
@@ -1,20 +1,16 @@
 import type { ExportData } from "./extractApi"
 import type { TsTagData } from "./tsDocTransforms"
 
-type KeywordData = {
-    scope: string
-    types: Record<string, Record<string, string>>
-}
-export const packDataForTable = (exportData: ExportData, tags: TsTagData) => {
-    const descriptions = tags.descriptions
-    const descriptionMatcher = /(?<=descriptions: ).+/
-    let descriptionObject: undefined | { [k: string]: string }
-    if (descriptions) {
-        const matchedDescription = descriptions[0]
+export const tabulateData = (exportData: ExportData, tags: TsTagData) => {
+    const keywords = tags.keywords
+    const keywordMatcher = /(?<=keywords: ).+/
+    let keywordsObject: undefined | { [k: string]: string }
```

Comment:
Avoid using data types like `Object` as a suffix, since we can see that from the TS anyways. `valuesByKey`, with values and key substituted for what each represent.

That would also be helpful in this case because it's not immediately clear what it is supposed to represent. You can also use `{[somethingSpecific: string]: string}` to help clarify this as well.

Last, best to put `| undefined` after the type since it's less important